src/common/ExportSettingsModal/LineExportSettingModalContainer.js

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { uniq } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { fetchAllRecords } from '@folio/stripes-acq-components';
+
+import {
+  ORDERS,
+  ORDER_LINES,
+} from '../../components/Utils/resources';
+import { fetchExportDataByIds } from './utils';
+import ExportSettingsModalContainer from './ExportSettingsModalContainer';
+
+const LineExportSettingsModalContainer = ({
```

Comment:
that component even in his name has domain - OrderLinesList. It's hard to say why you've decided to put it under common